addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentCard.js

```diff
@@ -31,7 +34,57 @@ class ScEnvironmentCard extends React.Component {
     return envType;
   }
 
+  getEnvTypeConfigsStore() {
+    const configsStore = this.envTypesStore.getEnvTypeConfigsStore(this.environment.envTypeId);
+    return configsStore;
+  }
+
+  getConfiguration(envTypeConfigId) {
+    const configsStore = this.getEnvTypeConfigsStore();
+    const config = configsStore.getEnvTypeConfig(envTypeConfigId);
+    return config;
+  }
+
+  getInstanceTypeFromConfigParams(config) {
+    const configParams = Object.entries(config.params);
+    const instanceTypeElement = _.findIndex(configParams, param => {
+      const key = Object.entries(param[1])[0][1];
```

Comment:
I originally was using the lodash `get`, but with the layering of the params (it's got multiple layers which are not very nice data structures which is why I was using the `Object.entries()` to make them things I could easily index on), the `get` doesn't actually GET the necessary information that is paired with the key.

As for the default, I went with "Not available" because the code broke the UI if I made it undefined or it didn't find the right parameter. I thought this was a graceful way to display that case (which should never happen in the wild because you can't make a working workspace without having the configuration have an instance type).